New preset for blackwidow keyboards

crimsonvspurple opened the pull request:
This preset uses 3 layouts.
1. Normal for quick casts
2. Space for self casts
3. ' (and M4) for smart casts

All 3 layouts have different chatwheels.

```
[ (M4) for shop
] (M2) for taunt
; (M3) for action item
```

```
Space+, (Space+M5) for mic toggle.
F5 for summon selection toggle.
F6 for right click repeat toggle.
J for HP marker.
```

All cycles have sound/texts for notifications.

M1-M5 functionalities are duplicated. This is because original blackwidow keyboards (2012) can not map M1-M5 to F13-F17.

To use this layout with those keyboards, go to Synapse and configure

```
M1 to [
M2 to ]
M3 to ;
M4 to '
M5 to ,
```

Additionally, you can configure M1 to a macro which presses [ and then v.

This will open this shop and highlight searchbox so you can directly type.

Commit:
New preset for blackwidow keyboards

dodekeract commented:
I'm not sure about the name of the preset. Many keyboards have a macro key row. I'd suggest changing it to something more personal, like your username or something which describes this layout in 2-3 words (if that's even possible).

When you change the name to something more descriptive I'm happy to merge this though. Couldn't test it since I'm not at home and only got my iPad Pro, but looks alright judging from the `json`. Only thing I noticed is that you're using the shop key binds to buy items. While this may work in this patch without any problems it will break if they add another item to that row. I suggest trying to set the stickybuy slot to the item, execute the buy sticky command and setting it back to TP scroll. That should be future proof and only needs a change if a user doesn't use TP scroll in sticky, which TBH is rare.

Also something seems to be misconfigured in your GitHub account, since the commits are made by a different user. You might want to check your git config's email-address and the ones known to GitHub.

crimsonvspurple committed:
Renamed the new preset and updated preload command

crimsonvspurple commented:
1. Renamed.
2. hud_sticky_item_name command doesn't work (or I cant get it to work)
3. Thanks. I don't use GitHub much. Fixed.

Other things:
4. force preload command has been changed in reborn; updated.
5. All of those multicore commands (except one) are invalid in reborn and the one available is a cheat. Should remove it altogether (have set false in this preset).

dodekeract commented:
Yeah I noticed that multicore issue in my logs already. Guess they just forced these things to be always on. Will see what I can do about that.

dodekeract commented:
Will update `npm` in some days. I will also see if I can improve the CLI. When I improved it you should be able to install this preset easily on any system.
